common/common.go

```diff
@@ -374,3 +381,27 @@ func RemapCurrencyCodeToSupported(code string) string {
 		return newCurrency
 	}
 }
+
+func MoveFile(sourcePath, destPath string) error {
```

Comment:
Reference for this method: https://gist.github.com/var23rav/23ae5d0d4d830aff886c3c970b8f6c6b

Reply:
mind leaving this as an actual comment in the code for future reference?